Gemfile.lock

```diff
@@ -19,6 +19,8 @@ GEM
     mustermann (3.0.0)
       ruby2_keywords (~> 0.0.1)
     nio4r (2.5.9)
+    nokogiri (1.15.2-arm64-darwin)
+      racc (~> 1.4)
```

Comment:
@nikolaTrichkovski these changes are not required for this specific task. Please remove changes in Gemfile.lock from this PR so we have a clean history